openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -214,32 +281,36 @@
         installHandlers: function() {
             var view = this;
             var $staffArea = $('.openassessment__staff-area', this.element);
-            var $staffTools = $('.openassessment__staff-tools', $staffArea);
-            var $staffInfo =  $('.openassessment__student-info', $staffArea);
+            var $manageLearnersTab = $('.openassessment__staff-tools', $staffArea);
             var $staffGradeTool = $('.openassessment__staff-grading', $staffArea);
 
             if ($staffArea.length <= 0) {
                 return;
             }
 
-            this.baseView.setUpCollapseExpand($staffTools, function() {});
```

Comment:
Note to reviewers: this was related to the old ui-toggle-visibility code. The method is still be used elsewhere, so it can't be deleted yet.